bootstrap4/templates/bootstrap4/index.js

```diff
@@ -20,6 +21,7 @@ import './inputTypes/number/number.html'
 import './inputTypes/password/password.html'
 import './inputTypes/radio/radio.js'
 import './inputTypes/range/range.html'
+import './inputTypes/range/range'
```

Comment:
Nice, now adding a range field! For a slider?

Reply:
Oh, this one is not intended. It must be removed or it may break the build process